school_transport/models/transport.py

```diff
@@ -124,6 +124,11 @@ def _search(self, args, offset=0, limit=None, order=None, count=False,
                                limit=limit, count=count,
                                access_rights_uid=access_rights_uid)
 
+    @api.multi
+    def set_over(self):
+        for rec in self:
+            rec.state = 'over'
```

Comment:
no need for loop as self.state will write on all ids. This is the advantage of write (set property)